docs/conf.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+project = 'astropy-helpers'
+copyright = '2019, The Astropy Developers'
```

Comment:
what's the policy on the year here? I mean it keeps coming up every year, and I keep forgetting. Should it be the full time period, or the latest or the first year?

Reply:
For now I'll fix it to make it match the license, but we can update this if we figure out what the right one is.

Reply:
Oh, sure. This was more like self mumbling that we need to figure out and act on separately across the whole ecosystem. (preferably with batchpr? 😉  )

Reply:
Ok cool :)